hudi-common/src/main/avro/HoodieMetadata.avsc

```diff
@@ -109,6 +109,14 @@
                                 "string"
                             ]
                         },
+                        {
+                            "doc": "Column name for which this column statistics applies",
```

Comment:
This change we'll have to revist when we tackle schema evolution. Can you please track it in a JIRA?

Reply:
In this case, we need an upgrade step if column stats index is enabled in metadata table, and this should be somewhat automatic.  @nsivabalan @vinothchandar wdyt?

Reply:
Why would write break? Addition of field is a valid schema evolution that we support right.
For reads, maybe we just handle this gracefully, if this field is not present in metadata table then fallback to usual query path (w/o data skipping).

Reply:
Not saying it would break, and based on the context provided it should be supported by schema evaluation, so I'm good with it.